From 629520a9cb77ff59188c82d126a274516e6e9405 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Tue, 4 Oct 2016 10:13:21 -0700
Subject: [PATCH] Disable systemd-resolved as default resolver.

Let docker, etc, use the direct nameserver information in
/etc/resolv.conf, instead of redirecting to the stub resolver in
systemd-resolved. The stub resolver fails on CNAME resolution and
breaks various applications.

Ref: https://github.com/coreos/coreos-overlay/pull/2129/commits/76830bedf02a8c54e1d2ce30796046b86bb00b94
---
 tmpfiles.d/etc.conf.m4 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tmpfiles.d/etc.conf.m4 b/tmpfiles.d/etc.conf.m4
index d88727e..12b61ff 100644
--- a/tmpfiles.d/etc.conf.m4
+++ b/tmpfiles.d/etc.conf.m4
@@ -12,6 +12,6 @@ m4_ifdef(`HAVE_SMACK_RUN_LABEL',
 t /etc/mtab - - - - security.SMACK64=_
 )m4_dnl
 m4_ifdef(`ENABLE_RESOLVED',
-L! /etc/resolv.conf - - - - ../usr/lib/systemd/resolv.conf
-L+! /etc/resolv.conf - - - - ../usr/lib/systemd/resolv.conf
+L! /etc/resolv.conf - - - - ../run/systemd/resolve/resolv.conf
+L+! /etc/resolv.conf - - - - ../run/systemd/resolve/resolv.conf
 )m4_dnl
-- 
2.10.0

